crates/secret-service-client/src/lib.rs

```diff
@@ -160,7 +160,7 @@ pub async fn make_v1_req(
     timeout_dur: Duration,
 ) -> Result<ServerMessage, ClientError> {
     let (mut tx, mut rx) = conn.open_bi().await.map_err(ClientError::ConnectionError)?;
-    let (len_bytes, msg_bytes) = VersionedClientMessage::V1(msg)
+    let (len_bytes, msg_bytes) = VersionedClientMessage::V1(msg.clone())
```

Comment:
I think `Message` is `Copy` no?
https://docs.rs/secp256k1/0.29.1/secp256k1/struct.Message.html